### PR TITLE
Fix/spanish&hmongantibiotics

### PR DESCRIPTION
**Pull request by TheJetro**
Finalized font streamlining for Spanish and Hmong antibiotics.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 25
-    buildToolsVersion '26.0.2'
```
Comment (mandemeskel) [inline app/build.gradle]: change this back to 26.0.2

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Tue Jan 30 09:43:53 PST 2018
+#Fri Apr 13 03:13:40 PDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```
Comment (mandemeskel) [inline gradle/wrapper/gradle-wrapper.properties]: revert this change as well

File: build.gradle [hunk 3]
```diff
@@ -3,9 +3,10 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'com.android.tools.build:gradle:3.1.1'
```
Comment (mandemeskel) [inline build.gradle]: revert the gradle version to 3.0.1

**Comment by TheJetro** [comment]
Es Done!
